{"name": "Проверяем что нельзя создать пользователя с уже занятым логином", "status": "passed", "description": "создаем двух пользователей, у второго меняем все кроме логина, проверяем ошибку", "start": 1721769782877, "stop": 1721769783477, "uuid": "52ec022a-4d63-462c-b17d-13690e8043bb", "historyId": "2f0512470973bab4b46cdef7052156a6", "testCaseId": "2f0512470973bab4b46cdef7052156a6", "fullName": "tests.test_create.TestCreate#test_create_same_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create"}, {"name": "subSuite", "value": "TestCreate"}, {"name": "host", "value": "MagnusEquusPC"}, {"name": "thread", "value": "10728-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create"}]}